files/master.cfg

```diff
@@ -31,12 +30,22 @@ class WorkerAdmins:
         except KeyError:
             return None
 
+{% endraw %}
+# Configurations controlled by ansible
```

Comment:
This comment is unnecessary. This is a jinja2 template so it's just redundant that it is rendered.

Reply:
ack